protoutil/blockutils.go

```diff
@@ -130,7 +138,7 @@ func GetMetadataFromBlockOrPanic(block *cb.Block, index cb.BlockMetadataIndex) *
 func GetConsenterMetadataFromBlock(block *cb.Block) (*cb.Metadata, error) {
 	m, err := GetMetadataFromBlock(block, cb.BlockMetadataIndex_SIGNATURES)
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to retrieve metadata at index: %d", cb.BlockMetadataIndex_SIGNATURES)
+		return nil, errors.Wrap(err, "failed to retrieve metadata")
```

Comment:
If you're changing this, let's do a `WithMessage` instead of a `Wrap`, no need to have two stacks attached.